src/test/java/org/zalando/baigan/service/FileSystemConfigurationRepositoryTest.java

```diff
@@ -12,7 +12,7 @@
 
     @Test
     public void testReadConfigurationsFromFile() {
-        final ConfigurationRepository repo = new FileSystemConfigurationRepository(3, "src/test/resources/example.json");
+        final ConfigurationRepository repo = new FileSystemConfigurationRepository("src/test/resources/example.json", 3);
```

Comment:
> Also I don't understand why the file repo needs a refresh

You can point the `FileSystemConfigurationRepository` to a file that you replace during runtime. I haven't used the repo though.

> Should this be 180 now because it changed from minutes to seconds?

Good catch, but the actual value doesn't make a difference for this test.